README.md

```diff
@@ -1,11 +1,95 @@
 # Capsule
 
-A framework for network function development. Written in Rust, inspired by [NetBricks](http://netbricks.io/) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
+A framework for network function development. Written in Rust, inspired by [NetBricks](https://www.usenix.org/system/files/conference/osdi16/osdi16-panda.pdf) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Quick Start](#quick-start)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [Contact](#contact)
+* [Maintainers](#maintainers)
+* [License](#license)
 
 ## Overview
 
-The project is currently undergoing heavy development. The goal is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for engineers. We are building a tool to efficiently manipulate network packets while being type-safe, memory-safe, and thread-safe.
+The goal of `Capsule` is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for engineers. We are building a tool to efficiently manipulate network packets while being type-safe, memory-safe, and thread-safe. By marrying `DPDK` with `Rust` Programming Language, `Capsule` offers
+
+* a fast packet processor that uses minimum number of CPU cycles.
+* a rich packet type system that guarantees memory-safety and thread-safety.
+* a concise framework that makes complex applications easy to write.
+
+## Quick Start
+
+The easiest way to get started developing `Capsule` applications is to use the `Vagrant` [virtual machine](https://github.com/capsule-rs/sandbox/blob/master/Vagrantfile) and the `Docker` [sandbox](https://hub.docker.com/repository/docker/getcapsule/sandbox) we provide. The sandbox is preconfigured with all the necessary tools and libraries for `Capsule` development.
+
+* [`DPDK` 18.11](https://doc.dpdk.org/guides-18.11/rel_notes/release_18_11.html)
```

Comment:
18.11.6 to be more specific here (LTS)?

Reply:
revisions are typically subject to change based on fixes. I think it's ok to not be exact here.

Reply:
ok. maybe link to the specific change set notes for .6?

Reply:
I think we want the whole change set. cuz it's all revisions up to `18.11.6`.